src/App.js

```diff
@@ -1,44 +1,22 @@
 
 import React, { Component } from 'react';
-
-import images from './data.js';
+import Header from './Header.js';
+import ImageList from './ImageList.js';
```

Comment:
looks good, nice work reaching the stretch and incorporating ImageList within your app.